docs/development.md

```diff
@@ -30,15 +30,6 @@ make dev-tools
 ### Services
 
 - [Kubernetes cluster](https://kubernetes.io). You can order a Kubernetes cluster on [ovh.com](https://www.ovh.com/fr/public-cloud/kubernetes/). Then configure your environment to have the right [`Kubeconfig`](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
-- [CertManager](https://cert-manager.io/docs/installation/kubernetes/#steps) >= 0.11
-
-  ```bash
-  helm install cert-manager \
-    --namespace cert-manager \
-    --version v0.12.0 \
-    jetstack/cert-manager
-  ```
```

Comment:
Would it be better to replace it with `kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml`

Reply:
I don't know if it's really relevant to do it here as it's already started in [Makefile](https://github.com/goharbor/harbor-operator/blob/master/Makefile#L450)

Reply:
What do you think ?

Reply:
I'm not quite sure if this area is worth modifying

Reply:
@thcdrt 

If want to let the developer use makefile to prepare the env, you can delete the section. But it seems we should explicitly mention the developer can use makefile to install prerequisite like cert-manager here. WDYT?

Reply:
Why not, we could redirect to the certmanager Makefile target, but in this case to be consistent we should talk of nginx ingress, postgresql-operator and redis-operator too, don't you think ?